cypress/e2e/shared/dashboardsIndex.test.ts

```diff
@@ -552,9 +548,14 @@ describe('Dashboards', () => {
       })
     })
   })
+  before(() =>
+    cy.exec('rm cypress/downloads/*', {
+      log: true,
+      failOnNonZeroExit: false,
+    })
+  )
 
-  // Skipping until https://github.com/influxdata/ui/issues/2864 is resolved by Bonitoo
-  it.skip('creates a dashboard and downloads JSON', () => {
+  it('creates a dashboard and downloads JSON', {browser: '!firefox'}, () => {
```

Comment:
why are we skipping this in firefox?

Reply:
I tried to solve the flaky behavior of the test. I managed to fix the behavior in Chrome, but Firefox kept failing from time to time. I changed the readfile several times to try out possible fixes, but none worked. Because there is no other way to ensure that the file downloaded, I decided to skip this test in Firefox to ensure it's stability. UI itself shouldn't be the problem, since I was unable to make the problem appear manually.

Reply:
So far as I can tell, there is no work around. Cy.readfile is the only option to check content of the cypress/downloads folder. Cy.fixture searches for files form the cypress/fixtures/ folder. 
The download process is handled internally in the browser (via blob). Because of that, checking the initiation of the download process is not an option eighter.
Also, there is not a notification which would indicate that the file downloaded successfully, like with the second export option (copy to clipboard).
In Firefox, the JSON file occasionally downloaded without it's content (NULL). Because of that, I added ".should('not.be.null')", to make the test fail due to a failure caused by the UI. Unfortunately, the problem persisted. When this error appears, cy.readfile is unable to read the mentioned file.
Similar problem appeared in Chrome, when I moved the removal of previously downloaded file to "beforeEach".
Putting it back in "before" fixed this behavior for Chrome.
I've been running the dashboardsIndex test for a few hours to provide you with some screenshots of the failure, but it hasn't crashed a single time. I will try to commit a version without excluding Firefox.